docs/PUBLISHING.md

```diff
@@ -42,43 +41,44 @@ yarn release
 # they're not represented in the commit log
 
 yarn release:major
+
+# Running this command will print instructions to complete the process. IGNORE THIS. Continue following the directions below.
 ```
 
-We use [standard-version](https://github.com/conventional-changelog/standard-version) to increment the version number in `package.json`, create a git tag for the new release, and update `CHANGELOG.md` based on the commit log. The package is not published, yet. If needed, you can [make additional changes to the CHANGELOG now](#editing-the-changelog).
+We currently use [standard-version](https://github.com/conventional-changelog/standard-version) to increment the version number in `package.json`, create a git tag for the new release, and update `CHANGELOG.md` based on the commit log. The package is not published, yet. If needed, you can [make additional changes to the CHANGELOG now](#editing-the-changelog).
 
-**NOTE**: running this command will print a command to run to complete the process. IGNORE THIS. Continue following the directions below.
+**NOTE**: `standard-version` is deprecated. Soon, we may move to an alternative script to handle releases.
 
 5. Push the release branch up, including tags:
 
 ```
-git push --follow-tags origin <branch>
+git push --follow-tags origin <branch> # this will also push tags
 ```
 
 #### Merging a release branch
 
-6. Open a pull request for the release branch, merging into **`main`**. 
+6. Open a pull request for the release branch, merging into **`main`**.
 
-For the commit message, use the new version's content in the [CHANGELOG.md](../CHANGELOG.md) (e.g., all the changes for version 9.0.0).
+For the commit message, use the new version's content in the [CHANGELOG.md](../CHANGELOG.md) (e.g., all the changes for version 9.0.0). Review the content in the changelog to make sure the notes, and the from-version and to-version are correct.
 
 Merge the PR through a **merge commit**:
-   ![github user interface showing a dropdown with the different merge options. the option "create a merge commit" is highlighted](https://user-images.githubusercontent.com/15840841/170514789-4f936ba2-c63d-486c-827a-b9e9e86b612e.png)
+
+![github user interface showing a dropdown with the different merge options. the option "create a merge commit" is highlighted](https://user-images.githubusercontent.com/15840841/170514789-4f936ba2-c63d-486c-827a-b9e9e86b612e.png)
```

Comment:
I think there's some discussion going on regarding the merge strategy / catching up but this is right as of now

Reply:
yep. this is the same, and just added a new line so that the image has a more legible layout